estate/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    'name': "My Awesome module Roole",
+    'version': '1.0',
+    'depends': ['base'],
+    'author': "roole",
```

Comment:
Fine in this tutorial but know that when developing an Odoo module, we must put "Odoo S.A.".